src/components/create-cards.jsx

```diff
@@ -1,9 +1,78 @@
 import React from 'react';
 
 class CreateCards extends React.Component {
-    render(){
-        return <h1 className="text-center">Create New Card</h1>
-    }
+  constructor(props) {
+    super(props);
+    this.state = {
+      question: '',
+      answer: ''
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(e) {
+    let value = e.target.value;
+    this.setState({
+      [e.target.name]: value
+    });
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    this.props.addCards(this.state);
```

Comment:
Here you want to create a new object from state not just pass state directly. This can cause undesired side effects.

EXAMPLE:
```JAVASCRIPT
this.props.addCards({...this.state});
```

In the example a new object is created from state and then passed into `addCard`